packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/vsphere-users/import/vsphere-user-import.html

```diff
@@ -0,0 +1,107 @@
+<div
```

Comment:
Please use a `oui-modal`

Reply:
It isn't compatible with data-wizard data-wizard-step.